src/.gitignore

```diff
@@ -0,0 +1,3 @@
+# generated by Happy
+Parser.hs
+Parser.info
```

Comment:
These shouldn't be generated outside `.stack-work` unless you run happy manually

Reply:
Ooh, makes sense.

Can't tell if this is just for my info, or an argument against having the .gitignore haha.

Reply:
I was kind of unsure as well, but I worry that a stale `Parser.hs` could cause trouble, so it might be better to leave it out so it's more easily noticeable.

Reply:
Ok I commented out the .gitignore line for Parser.hs